src/ar/calibration.cpp

```diff
@@ -106,19 +103,25 @@ static void help(char** argv)
         "     [-o=<out_camera_params>] # the output filename for intrinsic [and extrinsic] parameters\n"
         "     [-op]                    # write detected feature points\n"
         "     [-oe]                    # write extrinsic parameters\n"
+        "     [-oo]                    # write refined 3D object points\n"
         "     [-zt]                    # assume zero tangential distortion\n"
         "     [-a=<aspectRatio>]      # fix aspect ratio (fx/fy)\n"
         "     [-p]                     # fix the principal point at the center\n"
         "     [-v]                     # flip the captured images around the horizontal axis\n"
         "     [-V]                     # use a video file, and not an image list, uses\n"
         "                              # [input_data] string for the video file name\n"
         "     [-su]                    # show undistorted images after calibration\n"
+        "     [-ws=<number_of_pixel>]  # Half of search window for cornerSubPix (11 by default)\n"
+        "     [-dt=<distance>]         # actual distance between top-left and top-right corners of\n"
+        "                              # the calibration grid. If this parameter is specified, a more\n"
+        "                              # accurate calibration method will be used which may be better\n"
+        "                              # with inaccurate, roughly planar target.\n"
         "     [input_data]             # input data, one of the following:\n"
         "                              #  - text file with a list of the images of the board\n"
         "                              #    the text file can be generated with imagelist_creator\n"
         "                              #  - name of video file with a video of the board\n"
         "                              # if input_data not specified, a live view from the camera is used\n"
-        "\n", argv[0] );
```

Comment:
Why this change? Seems like a fine practice to use argv for this, in case we ever change the filename or name of the executable

Reply:
No idea, I just copied in their sample program basically. Not sure why they made that change but if it doesn't make sense we can just exclude it